analytics.go

```diff
@@ -332,17 +330,25 @@ func (c *Client) loop() {
 				c.verbose("interval reached – nothing to send")
 			}
 		case <-c.quit:
-			tick.Stop()
 			c.verbose("exit requested – draining msgs")
-			// drain the msg channel.
+
```

Comment:
This is a good point–any reason not to just set a flag on the client like `.closing` and check that pre-enqueue?